libs/agora/styles/src/_index.scss

```diff
@@ -1,5 +1,6 @@
-@use 'libs/shared/typescript/styles/src/index';
+// @use 'libs/shared/typescript/styles/src/index';
```

Comment:
moving away from shared monorepo styles and currently just using Agora specific styles

Reply:
This sounds good. The way we have setup the shared styles and themes is the main blocker for OC to migrate to esbuild. I believe that Tailwind CSS would provide a modern alternative.